clients/java/signalr/src/main/java/com/microsoft/signalr/OkHttpWebSocketWrapper.java

```diff
@@ -103,9 +103,7 @@ private void checkStartFailure() {
             // If the start task hasn't completed yet, then we need to complete it
             // exceptionally.
             if (!startSubject.hasComplete()) {
-                String errorMessage = "There was an error starting the Websockets transport.";
-                logger.error("Websocket closed from an error: {}.", errorMessage);
-                startSubject.onError(new RuntimeException(errorMessage));
+                startSubject.onError(new RuntimeException("There was an error starting the Websockets transport."));
```

Comment:
Casing and I prefer using the singular term (though we're inconsistent about that) since it is a transport that uses a WebSocket (not multiple WebSockets)

```suggestion
                startSubject.onError(new RuntimeException("There was an error starting the WebSocket transport."));
```